x/distribution/keeper/hooks.go

```diff
@@ -116,7 +125,6 @@ func (h Hooks) OnDelegationCreated(ctx sdk.Context, delAddr sdk.AccAddress, valA
 	h.k.onDelegationCreated(ctx, delAddr, valAddr)
```

Comment:
The new hook route should also be called within the `OnDelegationCreated` hook here

Reply:
I don't think there is a need now as I just moved the new call into `onValidatorModified`.